packages/dev-tools/src/events/EventIcon.tsx

```diff
@@ -1,21 +1,22 @@
 import { assertExhaustive } from "@latticexyz/common/utils";
-import { StoreEventsAbiItem } from "@latticexyz/store";
+import { StoreConfig } from "@latticexyz/store";
+import { StorageOperation } from "@latticexyz/store-sync";
 
 type Props = {
-  eventName: StoreEventsAbiItem["name"];
+  type: StorageOperation<StoreConfig>["type"];
 };
 
-export function EventIcon({ eventName }: Props) {
-  switch (eventName) {
-    case "StoreSetRecord":
+export function EventIcon({ type }: Props) {
+  switch (type) {
+    case "SetRecord":
       return <span className="text-green-500 font-bold">=</span>;
-    case "StoreSetField":
+    case "SetField":
       return <span className="text-cyan-500 font-bold">+</span>;
-    case "StoreDeleteRecord":
+    case "DeleteRecord":
       return <span className="text-red-500 font-bold">-</span>;
-    case "StoreEphemeralRecord":
-      return <span className="text-violet-400 font-bold">~</span>;
+    // case "EphemeralRecord":
+    //   return <span className="text-violet-400 font-bold">~</span>;
```

Comment:
is the plan to add `EphemeralRecord` as a separate `StorageOperation` or abstract it one level higher?

Reply:
I haven't decided 🙈 any opinions?